services/net/notification/appsettings.Development.json

```diff
@@ -6,6 +6,7 @@
       "TNO": "Debug"
     }
   },
+  "MmiaUrl": "https://mmia-dev.apps.silver.devops.gov.bc.ca",
```

Comment:
Move to the notification settings instead.   Not all notifications will need this value.

Reply:
Updated, thanks